src/test/java/UploadReceiptServletTest.java

```diff
@@ -75,17 +75,27 @@
       "com.google.sps.servlets.UploadReceiptServlet$FileNotSelectedException: No file was uploaded by the user (dev server).";
   private static final String INVALID_FILE_WARNING =
       "com.google.sps.servlets.UploadReceiptServlet$InvalidFileException: Uploaded file must be a JPEG image.";
+  private static final String INVALID_DATE_RANGE_WARNING =
+      "com.google.sps.servlets.UploadReceiptServlet$InvalidDateException: Transaction date must be in the past.";
+  private static final String INVALID_DATE_FORMAT_WARNING =
+      "com.google.sps.servlets.UploadReceiptServlet$InvalidDateException: Transaction date must be a long.";
   private static final String RECEIPT_ANALYSIS_FAILED_WARNING =
       "com.google.sps.servlets.ReceiptAnalysis$ReceiptAnalysisException: Receipt analysis failed.";
 
   private static final String INSTANT = "2020-06-22T10:15:30Z";
+  private static final long PAST_TIMESTAMP =
+      Instant.parse(INSTANT).minusMillis(1234).toEpochMilli();
+  private static final long FUTURE_TIMESTAMP =
```

Comment:
If this variable is only referenced in one test, I would just put it in that test method instead of as a class variable.